models/engine/file_storage.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+"""
+Module: file_storage.py
+"""
+import os
+
+class FileStorage():
+    """
+    serializes instances to a JSON file and
+    deserializes JSON file to instances
+    """
+    def __init__(self):
+        __file_path = "file.json"
+        __objects = {}
+
+    def all(self):
+        """
+        returns the dictionary __objects
+        """
+        return FileStorage__objects
+
+    def new(self):
+        """
+        sets in __objects the obj with key <obj class name>.id
+        """
+        key = "{}.{}".format(type(self).__name__, self.id)
+        FileStorage__objects[key] = __objects
+
+    def save(self):
+        """
+        serializes __objects to the JSON file (path: __file_path)
+        """
+        json.loads(__objects)
```

Comment:
Should be `json.dump(FileStorage.__object, FileStorage.__file_path)` to serialize the `__object` to the json file.

Reply:
Forgot to mention that the object instances in the `__object` dict should first be converted to  `dicts` before serializing to the json file. @cecilia-89